rfc.md

```diff
@@ -0,0 +1,322 @@
+# Friends Toy
+
+## Purpose
+This is a simple toy app designed to increase my familiarity with scala. 
+The basic purpose is to model friendships between people. 
+
+## Description
+For now all people will be held in a single "table" and will be self-joined, using a table that contains two user ids and a deleted flag in order to model relationships.
+We are also avoiding using an actual database (for now), choosing to focus initially on increasing familiarity with scala.
+The only initial piece of information we are holding onto about our "people" and "friends" is their names. For now we assume that names are unique.
+Upon creation, each person will be assigned a UUID that will be used to identify them.
+
+## Entities
+All entities should include timestamps for `created_at` and `last_modified`
+
+### People
+- `id`: pkey
```

Comment:
I would call these ids 'seq' as they do not serve the purpose of identifier (uuid is), but rather provide some temporal info.